### PR TITLE
Fixed bot access control

### PR DESCRIPTION
**Pull request by Lokosmos**

### CODE REVIEW

**Comment by matthew-morrison** [comment]
Can you provide a description for what this fixes?

**Comment by Lokosmos** [comment]
This patch fixes the problem of having a centralized control structure for the bot. By increasing the breadth of access to encompass TWO people instead of one, the problem is at least partially mitigated. I would accept an increase to three, but in my professional opinion four would be too many.

File: csss-minion.py [hunk 1]
```diff
@@ -108,6 +108,8 @@ def reloadConfig():
 def Henry(ctx):
     if ctx.message.author.id == "173702138122338305":
         return True
+    elif ctx.message.author.id == "140642264576819200":
```
Comment (Isvan) [inline csss-minion.py]: Yea... Dont think thats a good idea

**Comment by Isvan** [comment]
Its just giving him power to use the exe command and cogs

File: csss-minion.py (hunk 1 above)
Comment (brenfan) [inline csss-minion.py]: lololololol

**Pull request closed without merging**